api/task_processor/migrations/0008_add_get_task_to_process_function.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.18 on 2023-04-20 02:45
+
+from django.db import migrations
+
+from core.migration_helpers import PostgresOnlyRunSQL
+import os
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("task_processor", "0007_add_is_locked"),
+    ]
+
+    operations = [
+        PostgresOnlyRunSQL.from_sql_file(
+            os.path.join(
+                os.path.dirname(__file__),
+                "sql",
+                "get_tasks_to_process.sql",
+            ),
+            reverse_sql="DROP FUNCTION IF EXISTS get_tasks_to_process()",
```

Comment:
Having attempted to run this, I think the correct syntax is actually: 

```suggestion
            reverse_sql="DROP FUNCTION IF EXISTS get_tasks_to_process;",
```